packages/lab/src/__tests__/__e2e__/list-next/ListNext.cy.tsx

```diff
@@ -0,0 +1,346 @@
+            cy.mount(<SingleSelectList displayedItemCount={6} />);
+
+            cy.findByRole("listbox").then(($el) => {
+              expect($el.height()?.toFixed(0)).to.equal(
+                (3 * itemHeight).toString()
+              ); // 36px * 3
+            });
+          });
+        });
+      });
+
+      describe("WHEN disabled", () => {
+        describe("WHEN entire list is disabled", () => {
+          it("THEN does not allow any items to be selected", () => {
+            cy.mount(<SingleSelectList disabled />);
+
+            cy.findAllByRole("option").should(
+              "have.attr",
+              "aria-disabled",
+              "true"
+            );
+          });
+        });
+
+        describe("WHEN a specific list item is disabled", () => {
+          it("THEN it's not able to be selected", () => {
+            cy.mount(
+              <SingleSelectList
+                children={
+                  <>
+                    <ListItemNext value={ITEMS[0].value}>
+                      {ITEMS[0].label}
+                    </ListItemNext>
+                    <ListItemNext value={ITEMS[1].value} disabled>
+                      {ITEMS[1].label}
+                    </ListItemNext>
+                    <ListItemNext value={ITEMS[2].value}>
+                      {ITEMS[2].label}
+                    </ListItemNext>
+                  </>
+                }
+              />
+            );
+
+            cy.findByRole("option", { name: ITEMS[1].label }).should(
+              "have.attr",
+              "aria-disabled",
+              "true"
+            );
+          });
+
+          it("THEN it's not focusable on mouse hover", () => {
+            cy.mount(
+              <SingleSelectList
+                children={
+                  <>
+                    <ListItemNext value={ITEMS[0].value}>
+                      {ITEMS[0].label}
+                    </ListItemNext>
+                    <ListItemNext value={ITEMS[1].value} disabled>
+                      {ITEMS[1].label}
+                    </ListItemNext>
+                    <ListItemNext value={ITEMS[2].value}>
+                      {ITEMS[2].label}
+                    </ListItemNext>
+                  </>
+                }
+              />
+            );
+
+            cy.findByRole("option", { name: ITEMS[1].label })
+              .realHover()
+              .should("not.have.class", "saltListItemNext-focused");
+          });
+        });
+      });
+
+      describe("WHEN keyboard navigation and interactions", () => {
+        it("THEN select list item on Space or Enter key", () => {
+          cy.mount(<SingleSelectList />);
+
+          cy.findByRole("listbox").focus();
+          cy.realPress("ArrowDown");
+          cy.realPress("Space"); // select second list item
+          cy.findByRole("option", { name: ITEMS[1].label }).should(
+            "have.class",
+            "saltListItemNext-focused"
+          );
```

Comment:
Should this item also be selected? That's what the description says.
Would the second item be focused after ArrowDown?